src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -385,6 +389,8 @@ private static String suppressVariationsInInjectedValues(String content) {
     private static String replaceUnpredictableValuesWithPlaceholders(String content) {
         Date now = new Date();
         SimpleDateFormat sdf = new SimpleDateFormat("EEE, dd MMM yyyy, ");
+        // get session's time zone from content.
```

Comment:
Mention that this assumes only the page only shows time in one time zones, there are pages which can show multiple time zones (eg. InstructorSearchPage) and this method may not work on such pages.